nf_core/pipelines/create/templatefeatures.yml

```diff
@@ -242,3 +242,18 @@ changelog:
       - "CHANGELOG.md"
   nfcore_pipelines: False
   custom_pipelines: True
+license:
+  skippable_paths:
+    - "LICENSE"
+  short_description: "Add a license"
+  description: "Add the MIT license file."
```

Comment:
How about we add also a link to the requirement docs here?

Reply:
Which docs? 🤔

Reply:
https://nf-co.re/docs/guidelines/pipelines/requirements/mit_license

Reply:
ah ok! But this is only for nf-core pipelines, and the option to not add a license is for non-nf-core ones.